terraform/staging/main.tf

```diff
@@ -26,4 +26,24 @@ module "networking" {
   source = "../modules/networking"
 
   default_tags = var.default_tags
-}
\ No newline at end of file
+}
+
+resource "aws_key_pair" "bastion_ssh_key" {
+  key_name   = "bastion_ssh_key"
+  public_key = file("delta_test.pub")
+}
+
+module "bastion" {
+  source = "git::https://github.com/Softwire/terraform-bastion-host-aws?ref=34554d5b603e97d3adb2e06bcdf3b02d9d2a7e95"
```

Comment:
What's the `ref` param? I expected a commit hash or something. We definitely want to version it.

Reply:
It is a commit hash, just haven't truncated it

Reply:
I don't see any commits starting 3455: https://github.com/Softwire/terraform-bastion-host-aws/commits/master

Reply:
Ah, I see you have a branch on that repo. If you're making changes to it anyway, maybe use a tag?

Reply:
I'd rather leave it as a commit hash, even though it's a bit ugly it will always point to the same place so someone can't change it by pushing to the other repo. And yeah this points to a branch, but I'll merge it in at some point.